crates/gpui/src/action.rs

```diff
@@ -252,9 +252,10 @@ impl ActionRegistry {
 macro_rules! actions {
     ($namespace:path, [ $($name:ident),* $(,)? ]) => {
         $(
-            #[doc = "The `"]
+            // Unfortunately rust-analyzer doesn't display the name due to
+            // https://github.com/rust-lang/rust-analyzer/issues/8092
             #[doc = stringify!($name)]
-            #[doc = "` action, see [`gpui::actions!`]"]
+            #[doc = "action generated by `gpui::actions!`"]
```

Comment:
Did we lose a space here?

Seems like `action` would run together with the preceding action name.

Reply:
We didn't!  The macro expanded code is

```
        ///NoAction
        ///action generated by `gpui::actions!`
```

Reply:
Hmm, it seems like the `#[doc = stringify!($name)]` might not be doing anything, then?

Because when I hover an action all I see is this:

<img width="317" alt="Screenshot 2025-01-13 at 5 15 41 PM" src="https://github.com/user-attachments/assets/f3ac3630-b7c6-43cb-8cf0-a4fab9c22cff" />

And the action name is nowhere to be seen 🤔

Reply:
Correct!  That's what the comment this added is about

>             // Unfortunately rust-analyzer doesn't display the name due to